kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -3544,7 +3544,12 @@ object KyuubiConf {
       .version("1.8.0")
       .stringConf
       .toSequence()
-      .createWithDefault(Seq("spark.driver.memory", "spark.executor.memory"))
+      .createWithDefault(Seq(
+        "spark.driver.memory",
+        "spark.executor.memory",
+        "spark.kubernetes.context",
+        "spark.kubernetes.namespace",
+        "spark.master"))
```

Comment:
we don't need to expose those extra attrs as we decided to use `kyuubi.engine.appMgrInfo` to stored a base64 encoded string

Reply:
Got it. Thanks.